code/controllers/subsystem/shuttle.dm

```diff
@@ -69,13 +70,16 @@ SUBSYSTEM_DEF(shuttle)
 		supply_packs[P.type] = P
 		CHECK_TICK
 
-	// Initialize ftl13 station stocks
-	for(var/datum/star_system/system in SSstarmap.star_systems)
-		for(var/datum/planet/P in system.planets)
-			if(!P.station)
-				continue
-			P.station.generate()
-			CHECK_TICK
+// Initialize ftl13 station stocks/ station modules
+	for(var/i in SSstarmap.stations)
+		var/datum/space_station/station = i
```

Comment:
this one extra line of code must have been a lot of effort, feel free to take a break from coding after having to type this very long segment of code.